src/components/carousel/index.stories.mdx

```diff
@@ -0,0 +1,405 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+import Carousel from ".";
+import Image from "../image";
+import Button from "../button";
+import Icon from "../icon";
+
+export const SampleCard = ({ src }) => (
+	<div>
+		<h3 style={{ padding: "0 0.25rem" }}>
```

Comment:
While this is valid HTML - this isn't setting a great example of semantic and accessible HTML using `<h3>` here - Do we even need to use a heading here?